libs/imports/src/Imports.hs

```diff
@@ -4,7 +4,7 @@ module Imports
     -- * Base
       module Prelude
     , module Control.Applicative
-    , module Control.Monad
+    , module Control.Monad, whenM, unlessM
```

Comment:
Great, thanks for that!  Any reason why you didn't want to re-export the module?  (That would just re-export what's imported, no?)

Reply:
Those functions are not in `base`, please move them elsewhere in the export list.

Reply:
Also, thanks for adding them 🙂

Reply:
Ahh; didn't see the heading; just put them alongside the spot where I'd want them to be. I can move them.